src/apps/loyalty/client_side_analytics.ts

```diff
@@ -0,0 +1,3 @@
+import { initAnalytics } from "./analytics"
+
+initAnalytics()
```

Comment:
Ah as an entry point. I guess the `initAnalytics()` call could have gone into the same file as well, but I have no strong opinion on that.